realsense2_camera/src/profile_manager.cpp

```diff
@@ -198,8 +205,9 @@ VideoProfilesManager::VideoProfilesManager(std::shared_ptr<Parameters> parameter
     _module_name(module_name),
     _force_image_default_qos(force_image_default_qos)
 {
-    _allowed_formats[RS2_STREAM_DEPTH] = RS2_FORMAT_Z16;
-    _allowed_formats[RS2_STREAM_INFRARED] = RS2_FORMAT_Y8;
+    _allowed_formats[{RS2_STREAM_DEPTH, 0}] = RS2_FORMAT_Z16;
+    _allowed_formats[{RS2_STREAM_INFRARED, 0}] = RS2_FORMAT_RGB8;
```

Comment:
Can you explain the need for this?
It increase the buffer by ×3

Reply:
In D455 & D415:
Infra 0 stream supports           : RGB8, UYVY, RGBA8, etc.,
Infra 1 & 2 streams supports   : Y8, Y16, etc.,

There are no common formats which supports all 3 infra streams. So, have to declare separate default formats for each stream.